code/modules/power/supermatter/supermatter.dm

```diff
@@ -1124,6 +1124,13 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_crystal)
 		for(var/j in 1 to zap_count)
 			supermatter_zap(target, new_range, zap_str, targets_hit, zap_flags)
 
+/obj/machinery/power/supermatter_crystal/proc/destabilize(portal_numbers)
+	var/turf/T = get_turf(src)
+	if(T)
+		qdel(src)
+		new/obj/machinery/destabilized_crystal(T)
+		explosion(T,round(portal_numbers/10),round(portal_numbers/5),round(portal_numbers),1,1,1)
```

Comment:
This should probably not have an devestation tiles at all, considering how that completely negates the gas generation.

Reply:
true that, i'll set the devastation to zero